subprojects/docs/src/docs/userguide/publishingMaven.adoc

```diff
@@ -145,25 +145,17 @@ Maven restricts 'groupId' and 'artifactId' to a limited character set (`[A-Za-z0
 
 The only Unicode values that are explicitly prohibited are '```\```', '```/```' and any ISO control character. Supplied values are validated early in publication.
 
-[[sec:modifying_the_generated_pom]]
-==== Modifying the generated POM
+[[sec:customizing_the_generated_pom]]
+==== Customizing the generated POM
```

Comment:
I think this was an unnecessary change to the anchor. Any links to `#modifying_the_generated_pom` will no longer work. It's fine to change the section title if it has an explicit anchor, just not the anchor itself.

Reply:
Thanks for the feedback! I'll change it back.